### ISSUE TITLE
ECG Encoder Architectural Mismatch Between Paper and Code

### ISSUE THREAD

**Issue by willxxy** [AIoT-MLSys-Lab/MEIT#4]
Hello,

I was wondering if the following was known. In the paper of section 3.4 it states:

"Our default ECG encoder Fe(·) consists of temporal convolution blocks to encode the ECG recordings into embeddings. Specifically, each temporal convolution block comprises several 1-D convolution layers, batch normalization layers, and ReLU activation layers, followed by average pooling."

However, in the code of initializing the [ECG LLM](https://github.com/AIoT-MLSys-Lab/MEIT/blob/main/ECG_LLMs/ecg_llm.py#L36), 

```
class ECG_model(nn.Module):
    """
    ECG encoder model, includes ViT
    """
    def __init__(self, args=None, proj_hidden=2048, proj_out=128):
        super(ECG_model, self).__init__()
        # ecg signal encoder
        self.args = args
        ecg_model = args.ecg_model_type
        if ecg_model == 'ResNet18':
            self.ecg_encoder = ResNet18()
        elif ecg_model == 'ResNet18':
            self.ecg_encoder = ResNet34()
        elif ecg_model == 'ResNet50':
            self.ecg_encoder = ResNet50()
```

it seems to call the [Resnet architectures](https://github.com/AIoT-MLSys-Lab/MEIT/blob/main/ECG_LLMs/ecg_encoder.py). This would be quite a mismatch from the encoder described in the paper, thus I wanted to get any clarification on this!

Thank you,
William
